scalarm_client_csharp_app/ExampleSimulationScenario.cs

```diff
@@ -0,0 +1,45 @@
+using Scalarm.ExperimentInput;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Dynamic.Utils;
+
+namespace Scalarm
+{
+	public class ExampleSimulationScenario
+	{
+
+		public void Run()
+		{
+			var config = Application.ReadConfig ("config.json");
```

Comment:
Czy ten plik w ogóle coś robi, czy jest tylko roboczy do testów? Jeśli to drugie - nie commitujemy tego.